### ISSUE TITLE
Make helpers follow conventions

### ISSUE THREAD

**Issue by andysellick** [alphagov/govuk_publishing_components#3193]
According to the [helpers conventions](https://github.com/alphagov/govuk_publishing_components/blob/main/docs/component_conventions.md#helpers) we're not following the helpers conventions. Specifically:

- some applications are using presenters when they shouldn't (`finder-frontend`, `government-frontend` and `smart-answers` all use `shared_helper.rb`, for example)
- not all helpers are named `(component)_helper.rb` (consequence is they are not detected by the auditing)

Unfortunately to correct this would involve a number of breaking changes and it's not actually causing huge problems right now. Maybe this needs to begin with a discussion about these conventions, why they exist and how we should handle them from now on.

**Comment by jon-kirwan** [comment]
To be discussed at a future meeting to decide on way forward

**Comment by MartinJJones** [comment]
@andysellick I'm just wondering if this issue can be closed? it looks like quite a bit of work was done since the issue was original created to remove the use of the shared helper, for example - https://github.com/alphagov/finder-frontend/pull/3611

**Comment by andysellick** [comment]
Yeah good shout, I think I'm happy to close it. It's a bit vague - there's no specific real problem identified, and as you said, a lot of work has been done since it was logged (according to the auditing no applications are using the shared helper at all now).
